Add article on changing logo in Angular ABP apps

fahrigedik opened the pull request:
resolves  https://github.com/volosoft/lepton/issues/3106

<img width="1920" height="1080" alt="Adsız tasarım (13)" src="https://github.com/user-attachments/assets/47fe83bd-1356-43db-a442-aed283657c76" />

fahrigedik committed:
Add article on changing logo in Angular ABP apps

Introduces a new community article detailing how to configure and customize the application logo in Angular ABP applications. The article explains the migration of the logo provider to '@abp/ng.theme.shared', outlines the benefits of the new approach, and provides step-by-step implementation guidance.

sumeyyeKurtulus reviewed (commented):
Hello @fahrigedik thank you for writing such article to guide users 🚀. I can only recommend adding a section for [component replacement](https://abp.io/docs/latest/framework/ui/angular/component-replacement) as we can replace the logo component to customize it.

fahrigedik committed:
Enhance logo customization with component replacement

Added advanced customization options for logo component replacement in Angular ABP applications, including step-by-step instructions for implementation and a comparison with logo URL configuration.
